packages/ubuntu_bootstrap/lib/pages/source/codecs_and_drivers_page.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:ubuntu_bootstrap/l10n.dart';
+import 'package:ubuntu_bootstrap/pages/source/source_model.dart';
+import 'package:ubuntu_bootstrap/services.dart';
+import 'package:ubuntu_provision/ubuntu_provision.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru/yaru.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+
+export 'source_model.dart' show kFullSourceId, kMinimalSourceId;
+
+/// A page where the user can decide whether they want to install 3rd party
+/// drivers or codecs.
+class CodecsAndDriversPage extends ConsumerWidget with ProvisioningPage {
+  CodecsAndDriversPage({super.key});
+
+  final ScrollController _scrollController = ScrollController();
+
+  @override
+  Future<bool> load(BuildContext context, WidgetRef ref) {
+    return ref.read(sourceModelProvider).init().then((_) => true);
```

Comment:
@spydon something I missed before: we're calling `init()` again here - is this intentional?

Reply:
Ah, no, it's not! Maybe it should be split up into two models instead.
But maybe we'll just remove it for now to not spend time on it?

Reply:
Sorry, forgot about this today. I think it's simplest to keep the model intact and make sure to only call `init()` once. I'd prefer sticking to one model per directory in `pages/` - corresponding to a specific endpoint/controller in subiquity. If there are multiple pages in that directory the first one should initialize the model (or maybe we should make sure that subsequent `init()` calls don't have any effects).